tools/editor/scene_tree_dock.cpp

```diff
@@ -1617,8 +1615,9 @@ void SceneTreeDock::_normalize_drop(Node*& to_node, int &to_pos, int p_type) {
 	} else if (p_type==1) {
 			//drop at below selected node
 			if (to_node==EditorNode::get_singleton()->get_edited_scene()) {
-				to_node=NULL;
-				ERR_FAIL_COND(to_node==EditorNode::get_singleton()->get_edited_scene());
+				//if at lower sibling of root node
+				to_pos=0; //just insert at begining of root node
+				return;
```

Comment:
I've moved the code of the first  `if` on this CF block to here, so i could remove the unreachable code, even if it's not related with the issue.